app/frontend/shared/notifier.js

```diff
@@ -14,7 +14,7 @@ const notify = (title, text, type) => {
     notificationText =
       "<div>" +
       text +
-      "<div><button class='btn btn-secondary mt-4'>Refresh Page</button></div></div>";
+      "<div><button class='btn btn-secondary mt-4'>Reload Page</button></div></div>";
```

Comment:
I can't figure out why this was changed... 😅

Reply:
Updated the text in flash notification to `Reload Page` from `Refresh Page`, as we are using `reload the page` in every error texts.

Changed it to sound same 😅. This button is right below the error text.